index.js

```diff
@@ -0,0 +1,48 @@
+const checkLen = (password) => {
+  if (password.length >= 8) {
+    return true
+  } else {
+    return false
+  }
+}
+
+const checkLower = (password) => {
+  for (let i = 0; i < (password.length); i++) {
+    if (password[i] == password[i].toLowerCase() && password[i] != password[i].toUpperCase()) {
```

Comment:
You don't need both conditions, if it is equal to itself turned into lowercase then it cannot also be equal to itself turned into uppercase. A letter can't possibly be both.